jet_fuel/test/integration/url_usage_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+
+class UrlUsageTest < ActionDispatch::IntegrationTest
+  test "user enters a url and receives a shortened url" do
+    Capybara.current_driver = :selenium
+    sample_path = "http://www.laurawhalin.com"
+
+    visit root_path
+    fill_in("url[url]", with: sample_path)
+    click_link_or_button("Shorten")
+
+    shortened_url = Url.last.short_url
+
+    assert page.has_content?("Your shortened url is localhost:3000/#{shortened_url}")
+
+    click_link_or_button("short-url")
+
+    # assert_redirected_to sample_path
+  end
+
+  test "clicking a shortened url increases its click tally" do
+    sample_path = "http://www.laurawhalin.com"
+
+    visit root_path
+    fill_in("url[url]", with: sample_path)
+    click_link_or_button("Shorten")
+
+    shortened_url = Url.last.short_url
+    click_link('short-url', match: :first)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.